var/spack/repos/builtin/packages/tajo/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tajo(Package):
+    """
+    Tajo is a relational and distributed data warehouse system for Hadoop
+    Tajo is designed for low-latency and scalable ad-hoc queries, online
+    aggregation and ETL on large-data sets by leveraging advanced database
+    techniques. It supports SQL standards. It has its own query engine which
+    allows direct control of distributed execution and data flow. As a result,
+    Tajo has a variety of query evaluation strategies and more optimization
+    opportunities. In addition, Tajo will have a native columnar execution
+    and and its optimizer.
+    """
+
+    homepage = "http://tajo.apache.org/"
+    url      = "https://www-eu.apache.org/dist/tajo/tajo-0.11.3/tajo-0.11.3.tar.gz"
+    list_url = "https://www-eu.apache.org/dist/tajo/"
+    list_depth = 2
```

Comment:
I think this should be 1?

Reply:
Refine it in https://github.com/spack/spack/pull/14314/commits/bd536582c7c7c9e5185ee878140929dd4284064a